relay-server/src/statsd.rs

```diff
@@ -615,6 +625,8 @@ impl TimerMetric for RelayTimers {
             #[cfg(feature = "processing")]
             RelayTimers::StoreServiceDuration => "store.message.duration",
             RelayTimers::BufferInitialization => "buffer.initialization.duration",
+            RelayTimers::BufferIdle => "buffer.idle",
+            RelayTimers::BufferBusy => "buffer.busy",
```

Comment:
Could be the same metric using a tag, right?

Reply:
It could! But we can also sum up both metrics to get a full picture.